### ISSUE TITLE
how to do error message localization 

### ISSUE THREAD

**Issue by HosMercury** [Keats/validator#280]
how to deal with situations when i need the err msg to change depending on language?

**Comment by HosMercury** [comment]
how code related to the language ?

**Comment by Keats** [comment]
Think of the code as the key to your i18n translation map, eg `regex`.  Then you can use fluent of gettext to grab the relevant string for the current language code

**Comment by Keats** [comment]
No, `code` has nothing to do with HTML. It's something you can set for every validator in the derive (but built-in validator have a default code) like in the link in my first derive.
